src/main/java/chronosacaria/mcda/items/ArmorSetItem.java

```diff
@@ -58,18 +54,18 @@ public ArmorSetItem(ArmorSets set, EquipmentSlot slot) {
                     this.knockbackResistance, EntityAttributeModifier.Operation.ADDITION));
         }
 
-        StatBoosts statBoosts = config.statBoosts.get(set);
+        ArmorStats armorStats = config.armorStats.get(set);
 
         builder.put(EntityAttributes.GENERIC_ATTACK_DAMAGE, new EntityAttributeModifier(uuid,
-                "Armor attack damage " + boostString(statBoosts.attackDamage),
-                statBoosts.attackDamage,
+                "Armor attack damage boost",
+                armorStats.attackDamageBoost,
                 EntityAttributeModifier.Operation.MULTIPLY_BASE));
         builder.put(EntityAttributes.GENERIC_ATTACK_SPEED, new EntityAttributeModifier(uuid,
-                "Armor attack speed" + boostString(statBoosts.attackDamage),
-                statBoosts.attackSpeed, EntityAttributeModifier.Operation.MULTIPLY_BASE));
+                "Armor attack speed boost",
+                armorStats.attackSpeedBoost, EntityAttributeModifier.Operation.MULTIPLY_BASE));
         builder.put(EntityAttributes.GENERIC_MOVEMENT_SPEED, new EntityAttributeModifier(uuid,
-                "Armor movement speed" + boostString(statBoosts.attackDamage),
-                statBoosts.movementSpeed, EntityAttributeModifier.Operation.MULTIPLY_BASE));
+                "Armor movement speed boost",
+                armorStats.movementSpeedBoost, EntityAttributeModifier.Operation.MULTIPLY_BASE));
```

Comment:
Oh, whoops; I didn't notice that. Either way, I should have read the code more closely and approved rather than merged right away as you mentioned previously. I am still learning how to use Git in a manner that is conducive to communication and collaboration :P

I hadn't thought about the double negative aspect of the detriment. The main reason why I distinguished was because I could not get the config to accept a negative value so I made it a detriment and left if out from the config. I see what you mean though; and if I still leave it from the config, there is no reason why to not just assign the negative value to the boost, using the same nomenclature and not adding new things for the programme to keep track of. I hope that I am following you correctly.

Reply:
> The main reason why I distinguished was because I could not get the config to accept a negative value

Huh, that should be no issue at all.

I think what may have happened is that your config already had some value (like 0), you then changed the default values to something negative - but when launching the game it loaded the 0 instead of the new default value (which is the issue I mentioned at the end of the pull request).

> and if I still leave it from the config, there is no reason why to not just assign the negative value to the boost,
> ...
> I am following you correctly.

I'm not sure **I** am following you correctly, why "if I still leave it from the config"?

Currently (even in the previous refactor), the "detriments" are simply saved as negative boosts, e.g.:
```
		"PLATE": { 
			...
			"movementSpeedBoost": -0.0375
		},
```

Reply:
Sorry, my English was kind of bad in that last comment. Sorry about that. I was trying to say that I had hardcoded the detriment because I could not get the negative value to take in the config; but clearly what you did allowed it to work. I knew that it not accepting the value had to be user error on my part. Sorry for the confusion.